packages/edit-site/src/components/page-patterns/style.scss

```diff
@@ -6,14 +6,6 @@
 	padding: 0;
 	overflow-x: auto;
 
-	.edit-site-page-content {
-		height: 100%;
-		position: relative;
-		padding: 0;
-		display: flex;
-		flex-flow: column;
-	}
```

Comment:
I have defined this as the default style [here](https://github.com/WordPress/gutenberg/pull/53207/files#diff-faf245a8ddb49305b5d7b7c5ce3ca8b5c5a973a0feaed2112fa3d8a87366a981R34-R38), so overriding this style for Pattern page only is no longer necessary.